### PR TITLE
fix(ui): Add project id to release link, center icons

### PR DESCRIPTION
**Pull request by matejminar**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx [hunk 1]
```diff
@@ -169,11 +169,10 @@ class GroupActivity extends React.Component {
                       date={item.dateCreated}
                       header={
                         <GroupActivityItem
-                          organization={organization}
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx:172]: organization and projectId (slug) were not being used

Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx:172]: :+1: Didn't find any spreads in this component either.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx:172]: yeah, me neither 🙂

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx [hunk 2]
```diff
@@ -169,11 +169,10 @@ class GroupActivity extends React.Component {
...
                           author={<ActivityAuthor>{authorName}</ActivityAuthor>}
                           item={item}
-                          orgId={this.props.params.orgId}
-                          projectId={group.project.slug}
+                          orgSlug={this.props.params.orgId}
+                          projectId={group.project.id}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx:175]: 😅

**Review by billyvg — APPROVED** [review]

**Pull request merged**